plugins/check_http.c

```diff
@@ -57,8 +58,8 @@ enum {
 };
 
 #ifdef HAVE_SSL
-int check_cert = FALSE;
-int continue_after_check_cert = FALSE;
+bool check_cert = false;
```

Comment:
what's wrong with FALSE?

Reply:
IMHO, booleans should be "real" booleans, not aliased integers. Sure, it might be the same in the end, but this gives the compiler and future developers more context about what this is doing and what could/should possibly happen there.

I personally find it hugely confusing if an `int` variable or function turns out to be used for booleans only.

Reply:
ok, agreed